runtime/interpreter/big.go

```diff
@@ -57,19 +57,55 @@ func SignedBigIntToBigEndianBytes(bigInt *big.Int) []byte {
 	}
 }
 
-func UnsignedBigIntToBigEndianBytes(bigInt *big.Int) []byte {
+func SignedBigIntToSizedBigEndianBytes(bigInt *big.Int, sizeInBytes uint) []byte {
+	// todo use uint64 for fewer iterations?
+	buf := make([]byte, sizeInBytes)
 
 	switch bigInt.Sign() {
 	case -1:
+		increm := big.NewInt(0)
+		increm = increm.Add(bigInt, bigOne)
+		bytes := increm.Bytes()
+		offset := len(buf) - len(bytes)
+		for i := 0; i < offset; i++ {
+			buf[i] = 255 // sign extend
+		}
+		for i := 0; i < len(buf)-offset; i++ {
+			buf[i+offset] = ^bytes[i]
+		}
+	case 0:
+		break
+	case 1:
+		bigInt.FillBytes(buf)
+	default:
```

Comment:
codecov is yelling about these unreachable cases but how do we even test them? mock big.Int and add a pathological .Sign() implementation?

Reply:
No, we don't test those, they're just a fallback